flocker/control/test/test_script.py

```diff
@@ -0,0 +1,54 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+from twisted.web.server import Site
+from twisted.trial.unittest import SynchronousTestCase
+
+from ..script import ControlOptions, ControlScript
+from ...testtools import MemoryCoreReactor, StandardOptionsTestsMixin
+
+
+class ControlOptionsTests(StandardOptionsTestsMixin,
+                          SynchronousTestCase):
+    """
+    Tests for ``ControlOptions``.
+    """
+    options = ControlOptions
+
+    def test_default_port(self):
+        """
+        The default port configured by ``ControlOptions`` is 4523.
+        """
+        options = ControlOptions()
+        options.parseOptions([])
+        self.assertEqual(options["port"], 4523)
+
+    def test_custom_port(self):
+        """
+        The ``--port`` command-line option allows configuring the port.
+        """
+        options = ControlOptions()
+        options.parseOptions(["--port", 1234])
```

Comment:
Guess that was a pre-existing defect since this code is just being moved.  Still probably sensible to fix it now though.